docs/manage-metadata-using-gravitino.md

```diff
@@ -733,6 +733,159 @@ In addition to the basic settings, Gravitino supports the following features:
 | Bucketed table      | Equal to `CLUSTERED BY` in Apache Hive, some engine may use different words to describe it.  | [Distribution](pathname:///docs/0.3.0/api/java/com/datastrato/gravitino/rel/expressions/distributions/Distribution.html) |
 | Sorted order table  | Equal to `SORTED BY` in Apache Hive, some engine may use different words to describe it.     | [SortOrder](pathname:///docs/0.3.0/api/java/com/datastrato/gravitino/rel/expressions/sorts/SortOrder.html)               |
 
+#### Partitioned table
+
+Currently, Gravitino supports the following partitioning strategies:
+
+:::note
+The `score`, `dt` and `city` are the field names in the table.
```

Comment:
How about adding more modifications like:
The `score`, `dt`, and `city` appearing in the table below refer to the field names in a table.

Reply:
Nice suggestion.